smoke-tests/src/test/groovy/org/hypertrace/agent/smoketest/GlassFishSmokeTest.groovy

```diff
@@ -1,4 +1,45 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
 package org.hypertrace.agent.smoketest
 
-class GlassfishSmokeTest {
+import spock.lang.Ignore
+
+import java.time.Duration
+import org.testcontainers.containers.wait.strategy.Wait
+import org.testcontainers.containers.wait.strategy.WaitStrategy
+
+@Ignore
```

Comment:
please create an issue and add a link here

Reply:
https://github.com/hypertrace/javaagent/issues/276